src/zygote_forwarddiff.jl

```diff
@@ -1,68 +1,20 @@
 # Zygote loads ForwardDiff, so this adjoint will autmatically be loaded together
 # with `using Zygote`.
 
-# FIXME: This is inefficient, replace with the commented code below once Zygote supports it.
+# TODO: add adjoints without ForwardDiff
 @adjoint function poissonbinomial_pdf_fft(x::AbstractArray{T}) where T<:Real
     fft = poissonbinomial_pdf_fft(x)
     return  fft, Δ -> begin
-        ((ForwardDiff.jacobian(x -> poissonbinomial_pdf_fft(x), x)::Matrix{T})' * Δ,)
+        ((ForwardDiff.jacobian(poissonbinomial_pdf_fft, x)::Matrix{T})' * Δ,)
     end
 end
 
-# The code below doesn't work because of bugs in Zygote. The above is inefficient.
```

Comment:
why remove this?

Reply:
Because the FFT implementation is slow and not used anymore in Distributions. It is just kept there (for now) to be able to compare and test the new faster implementations. So the whole FFT-based implementations can be removed in the near future.

Additionally, if needed, the git history contains all the comments.